modules/farm_fd2_school/src/entrypoints/Vue2/App.vue

```diff
@@ -115,6 +126,12 @@ export default {
         { id: 4, date: '06/05/2019', location: 'GHANA', bed: 'GHANA-4' },
       ],
       unitList: ['BUNCH', 'EACH', 'POUND'],
+      date: '2019-06-15',
+      crop: 'RADISH',
+      quantity: 1,
+      unit: 'BUNCH',
+      comment: '',
+      plantIndex: '',
```

Comment:
Using `''` for the initially selected radio button works. But it may be more consistent to use `-1`, which is a common convention for values that are invalid indices (e.g. many searching algorithms will return `-1` when the value being searched for is not found).